agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/status/StatusFile.java

```diff
@@ -139,7 +132,7 @@ static String initLogDir() {
     if (homeDir != null && !homeDir.isEmpty()) {
       return homeDir + DEFAULT_LOGDIR + DEFAULT_APPLICATIONINSIGHTS_LOGDIR;
     }
-    return DEFAULT_HOME_DIR + DEFAULT_LOGDIR + DEFAULT_APPLICATIONINSIGHTS_LOGDIR;
+    return DiagnosticsHelper.isOsWindows() ? WINDOWS_DEFAULT_HOME_DIR : LINUX_DEFAULT;
```

Comment:
do you know when HOME env var is/isn't set? do we want to ignore HOME env var on linux?

Reply:
HOME env var has the highest precedence. it's the first if statement. if it has value, it returns right away. it wouldn't reach line 135.

Reply:
it's considered for both linux and windows.

Reply:
I thought for linux it needed to be under `/var/log/applicationinsights`?

Reply:
it is. you're asking home env var, which is for windows.

Reply:
it looks to me like the code will also pick up HOME env var on linux, and then it won't end up in `/var/log/applicationinsights`

Reply:
home env var is not used by linux as far as i understand. i just reverted back to what it was. if you have concern, we can add isWindows? but if linux also uses this env var, we should respect that, don't you think?